modules/custom/az_news/config/install/core.entity_view_display.node.az_news.default.yml

```diff
@@ -115,7 +115,7 @@ third_party_settings:
       weight: 6
       format_type: html_element
       format_settings:
-        classes: 'col-lg-10 offset-lg-1'
+        classes: 'col-12'
```

Comment:
```suggestion
        classes: 'col'
```

Reply:
Matching previously used settings and we do want to force it into col-12 rather than having it be flexible despite being only one column.